template/topup_request.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Your topup request has been received.</h1>
```

Comment:
How is this view called? not saying its wrong, just not sure why this template is needed.

Reply:
So the user clicks on a Whatsapp link first which then shows this template letting the contact know that their top-up request has been received and then we fire a POST to the endpoint to start the flow. So this view is just for user experience.

Reply:
https://drive.google.com/file/d/1hxLEkEwxdPtDr3pWX-DK9_YonrB3w0PJ/view?usp=sharing